examples/ecs/error_handling.rs

```diff
@@ -55,7 +55,7 @@ fn main() {
 
     // If we run the app, we'll see the following output at startup:
     //
-    //  WARN Encountered an error in system `fallible_systems::failing_system`: "Resource not initialized"
+    //  WARN Encountered an error in system `fallible_systems::failing_system`: Resource not initialized
```

Comment:
I changed this line of documentation because we'll no longer print the quotes, but the example doesn't actually output this error!  The sole use of `failing_system` is used with `pipe` to handle the result.